### ISSUE TITLE
Too long timeout after disconnection

### ISSUE THREAD

**Issue by ifyousayso** [elysium-project/server#1776]
When a character disconnects in any way, it should be saved and removed from the world after exactly 1 minute, even if it's in combat or any other non-parked situation.. like falling.

Yes, this allowed for certain abuse, but the ridiculously long time that is used now is also abusable, so fix it or don't fix it..

**Comment by stesve** [comment]
If im not mistaken you are currently getting "removed" 2 mins after a dc. That seems like the correct value to me.

**Comment by ifyousayso** [comment]
You are mistaken because, at least on Earthen Ring in 2006, when the raid was about to fail, one could ALT+F4, wait one minute (while communicating with others in game about the boss reset and status of one's character) then log back on and resurrect the raid. The others confirmed that one's character was removed after one minute. Perhaps it was 2 minutes on other realms..

Please don't implement what seems to be right, because on top of the bugs that existed, there were also more or less weird design choices.

**Comment by stesve** [comment]
It doesn't matter if you have lost your connection or not. During a bossfight there is a raidwide pulse that puts everyone in combat, regardless of their connectionstate.

**Comment by ifyousayso** [comment]
1. Irrelevant. This issue has nothing to do with the combat state. I was merely pointing out that one should be removed from the game even if in combat.
2. False. Not all raid bosses put you in combat, but I skip the positive bug reports until once all the bad ones have been fixed. It's not fair that all should waste the player's time and character's gold..

**Comment by stesve** [comment]
1 - they are, but after 2 mins.
2 - all raidbosses puts you in combat. There was a raidwide pulse recently implemented to reflect how it worked on retail vanilla.

**Comment by ifyousayso** [comment]
Perhaps that implementation was active yesterday, when you posted, but it doesn't work now.. on Darrowshire.

I am reporting bugs. You believe whatever you want to.
